app/helpers/get-ancestor-descriptor.js

```diff
@@ -1,3 +1,4 @@
+import ObjectProxy from '@ember/object/proxy';
```

Comment:
Oh hey, I didn't know this helper existed. We should consolidate it with [this](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/components/dashboard-item/component.ts#L17). (Sometime. Not in this PR).